shared/src/dateHelpers.ts

```diff
@@ -21,12 +21,12 @@ export const getCurrentDateInNewJersey = (): Dayjs => {
   return dayjs().tz("America/New_York");
 };
 
-export const getCurrentDateInNewJerseyISOFormatted = (): string => {
-  return getCurrentDateInNewJersey().toISOString();
+export const getCurrentDateFormatted = (format: string): string => {
+  return getCurrentDate().format(format);
```

Comment:
Nice catch!

Reply:
[dust] Question for @jphechter: is there ever a situation where we want to `getCurrentDate` without getting the current date in New Jersey?

Reply:
I'm sure, for instance something that has to do with handling data server-side. I don't see a reason that we'd want to calculate for NJ specifically, even if they do happen to be the same. I didn't want to completely replace for that reason.